WordPress/Classes/ViewRelated/Post/Scheduling/SchedulingCalendarViewController.swift

```diff
@@ -0,0 +1,200 @@
+import Foundation
+import Gridicons
+
+protocol DateCoordinatorHandler: class {
+    var coordinator: DateCoordinator? { get set }
+}
+
+struct DateCoordinator {
```

Comment:
Because this is injected in trough the view hierarchy I would make it a class to avoid the copy and use it as reference instead.